application/performance/locust/locustfile.py

```diff
@@ -0,0 +1,138 @@
+import json
+from locust import task, FastHttpUser, tag, LoadTestShape, constant_pacing
+import config as config
+
+
+class FiveDest(FastHttpUser):
+    weight = 80
+
+    def on_start(self):
+        with open(config.ccs_prefix + 'ccs_5_destinations.json') as json_file:
+            self.payload = json.load(json_file)
+
+    @tag('load')
+    @task
+    def start_test(self):
+        self.client.post("", data=json.dumps(self.payload), headers=config.headers)
+
+
+class FiftyDest(FastHttpUser):
+    weight = 15
+
+    def on_start(self):
+        with open(config.ccs_prefix + 'ccs_50_destinations.json') as json_file:
+            self.payload = json.load(json_file)
+
+    @tag('load')
+    @task
+    def start_test(self):
+        self.client.post("", data=json.dumps(self.payload), headers=config.headers)
+
+
+class FiveHundredDest(FastHttpUser):
+    weight = 3
+
+    def on_start(self):
+        with open(config.ccs_prefix + 'ccs_500_destinations.json') as json_file:
+            self.payload = json.load(json_file)
+
+    @tag('load')
+    @task
+    def start_test(self):
+        self.client.post("", data=json.dumps(self.payload), headers=config.headers)
+
+
+class OneThousandFiveHundredDest(FastHttpUser):
+    weight = 1
+
+    def on_start(self):
+        with open(config.ccs_prefix + 'ccs_1500_destinations.json') as json_file:
+            self.payload = json.load(json_file)
+
+    @tag('load')
+    @task
+    def start_test(self):
+        self.client.post("", data=json.dumps(self.payload), headers=config.headers)
+
+
+class ThreeThousandDest(FastHttpUser):
+    weight = 1
+
+    def on_start(self):
+        with open(config.ccs_prefix + "ccs_3000_destinations.json") as json_file:
+            self.payload = json.load(json_file)
+
+    @tag('load')
+    @task
+    def start_test(self):
+        self.client.post("", data=json.dumps(self.payload), headers=config.headers)
+
+
+class ColdStartFiveDest(FastHttpUser):
+    weight = 80
+    wait_time = constant_pacing(300)
```

Comment:
This point refers to each of the ColdStartClasses
Is this wait_time mean that there is always 300s between requests?
The scenario asks for an increasing wait between each request, should it increase by 60 secs between each request to understand the point at which the cold start kicks in?